Flow.Launcher.Infrastructure/Http/Http.cs

```diff
@@ -45,6 +46,7 @@ public static HttpProxy Proxy
             {
                 proxy = value;
                 proxy.PropertyChanged += UpdateProxy;
+                UpdateProxy(ProxyProperty.Enabled);
```

Comment:
Manually trigger an update. Otherwise, `WebProxy` would be `null` until you make a change in the settings.

Reply:
Great fix! Thank you!
Is it possible that the SocketHttpHandler doesn't work is because of this issue?

Reply:
I don't think so. It still does not work with all my other changes.